src/Appwrite/Utopia/Response/Model/AlgoArgon2.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Appwrite\Utopia\Response\Model;
+
+use Appwrite\Utopia\Response;
+use Appwrite\Utopia\Response\Model;
+
+class AlgoArgon2 extends Model
+{
+    public function __construct()
+    {
+        // No options if imported. If hashed by Appwrite, following configuration is available:
+        $this
+            ->addRule('memoryCost', [
+                'type' => self::TYPE_INTEGER,
+                'description' => 'Memory that may be used to compute hash.',
+                'default' => '',
+                'example' => 65536,
+            ])
+            ->addRule('timeCost', [
+                'type' => self::TYPE_INTEGER,
+                'description' => 'Amount of time it may take to compute hash.',
```

Comment:
```suggestion
                'description' => 'Amount of time it took to compute hash.',
```

Reply:
I dislike the verb `take` here, it's used mostly informally to describe the passage of time.

> Amount of time consumed to compute hash

Reply:
Updated my suggestion.